templates/express/server.js

```diff
@@ -48,31 +51,77 @@ const port = process.env.PORT || 3000;
 app.listen(port, async () => {
   console.log(`Express server listening on port ${port}`);
 
+  // send "ready" message to dev server
   if (process.env.NODE_ENV === "development") {
     broadcastDevReady(build);
   }
 });
 
+
+// Create a request handler that watches for changes to the server build during development.
 function createDevRequestHandler() {
-  const watcher = chokidar.watch(BUILD_PATH, { ignoreInitial: true });
+  async function handleServerUpdate() {
+    // 1. re-import the server build
+    build = await reimportServer();
 
-  watcher.on("all", async () => {
-    // 1. purge require cache && load updated server build
-    const stat = fs.statSync(BUILD_PATH);
-    build = import(BUILD_PATH + "?t=" + stat.mtimeMs);
-    // 2. tell dev server that this app server is now ready
-    broadcastDevReady(await build);
-  });
+    // Add debugger to assist in v2 dev debugging
+    if (build?.assets === undefined) {
+      console.log(build.assets);
+      debugger;
+    }
 
+    // 2. tell dev server that this app server is now up-to-date and ready
+    broadcastDevReady(build);
+  }
+
+  chokidar
```

Comment:
`chokidar` isn't a production dependency, so it would be better to use `(await import("chokidar"))`, remove the `chokidar` import at the top of the file, make `createDevRequestHandler()` async, and await on it at the `app.all()` handler.

Reply:
Updated the patch to use await import

Reply:
In my code, I imported directly in the `createDevRequestHandler()` function, but this way is probably the cleanest if swapping in CJS is important.